docs/openapi.yaml

```diff
@@ -10,6 +10,8 @@ x-amazon-apigateway-request-validators:
 paths:
   /candidate:
     get:
+      tags:
+        - Candidate
```

Comment:
Tags wont work when we merge docs together. I suggest removing them
https://swagger-ui-internal.dev.nva.aws.unit.no/

Reply:
Removed